### PR TITLE
LUIS can be used via a str preset now && replacing word vectors is now a preset

### LINKED ISSUE #129: Add str for LUIS preset
- [x] should be able to use it without args, like other presets

### PR DESCRIPTION
**Pull request by michaelperel**
fixes #129 

default behavior for str:
 * each token becomes an entity
 * The first group in each utterance becomes the intent

If intents or entities are specified via the preset, only those intents and entities will be used.

### CODE REVIEW

**Comment by michaelperel** [comment]
closes #133 as well

**Comment by michaelperel** [comment]
ready for review

**Comment by michaelperel** [comment]
Also adds a preset for stochastic.

closes #131 .

Ready for review

File: putput/presets/stochastic.py [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+        ...     print(tokens)
+        ...     print(groups)
+        ...     break
+        can she steal curry can herself get fries whilst fries
+        ('[ADD(can she steal)]', '[ITEM(curry)]', '[ADD(can herself get)]',
+         '[ITEM(fries)]', '[CONJUNCTION(whilst)]', '[ITEM(fries)]')
+        ('{ADD_ITEM([ADD(can she steal)] [ITEM(curry)])}', '{ADD_ITEM([ADD(can herself get)] [ITEM(fries)])}',
+         '{None([CONJUNCTION(whilst)])}', '{None([ITEM(fries)])}')
+    """
+    chance = min(max(chance, 0), 100)
```
Comment (armanrahman22) [inline putput/presets/stochastic.py]: can we make this throw an error instead?

**Pull request merged**